packages/moonstone/Panels/Header.module.less

```diff
@@ -155,6 +155,12 @@
 		.padding-start-end(0, var(--moon-panels-controls-width));
 	}
 
+	.header.centered.compact,
```

Comment:
I don't think compact should be included here. Compact Header isn't intended to be centered and would look weird if it were, so I don't think this is necessary.

Reply:
I've attached screenshot on compact header in the ticket.